### DISCUSSION TITLE
Snapclient player options

### DISCUSSION THREAD

**Discussion by artibea** [snapcast/snapcast#1223] [General]
Hello.

I'm struggling to make snapclient working on raspberry pi with Debian Bullseye (Bookwarm also) and bluetooth speakers.
After adding "--player pulse" in SNAPCLIENT_OPTS service failing to start.

Command $ snapclient --player pulse - works and can listen music through bluetooth speakers. 

How I can tell snapclient to always use pulse player in systemd? Alsa dosesn't work with bluetooth.

Thanks for any ideas.

**Comment by seppeel** [comment]
The place were you can add startoptions to your systemd service is
`/etc/default/snapclient`

My example:
```
$ cat /etc/default/snapclient
## Start the client, used only by the init.d script
START_SNAPCLIENT=true

# Additional command line options that will be passed to snapclient
# note that user/group should be configured in the init.d script or the systemd unit file
# For a list of available options, invoke "snapclient --help"
SNAPCLIENT_OPTS="-h <ip> --player pulse"
```
this uses pulseaudio and directly connects the client to the snapserver with `<ip>`
